custom_components/tuya_local/devices/smart_powerstrip_3_outlet_energy.yaml

```diff
@@ -0,0 +1,154 @@
+name: 3 Outlet Smart Powerstrip + Energy monitor
```

Comment:
Not a fan of names starting with numbers, and non alpha-numeric characters like + in names, because these names get transformed into entity ids within Home Assistant, and I'm not sure if these things might cause issues in some contexts. Also, the entity name get appended to this, so it shouldn't be too long.

My suggestion: "Triple monitoring powerstrip"

Reply:
I agree on this change, starting with numbers can be a problem in some cases, and special characters are not good friends with ids. Your suggestion covers perfectly the previous name meaning